### PR TITLE
remove test_suite, add pytest as test_requires

### PR DESCRIPTION
**Pull request by mikaelarguedas**
connects to https://github.com/ros2/ros2/issues/414#issuecomment-344103560
Feel free to close if this was not changed on purpose

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Should the Travis file use `pytest` then instead of `nose`?

File: .travis.yml [hunk 1]
```diff
@@ -28,6 +28,6 @@ install:
   - python setup.py install
   - cd ../..
 script:
-  - python setup.py nosetests -s
+  - python setup.py test
```
Comment (dirk-thomas) [inline .travis.yml]: This doesn't work. See the latest Travis log which says "Ran 0 tests".

Comment (mikaelarguedas) [reply .travis.yml]: This is working locally but not on travis. Will debug tomorrow and leave it in progress for now

**Comment by mikaelarguedas** [comment]
Looks like pytest doesnt like test files with the same name in the same setup.py invocation.
I'm going to close this for now as I don't have time to look into it and that nose is fine with it

**Pull request closed without merging**